include/mfmg/common/amge.templates.hpp

```diff
@@ -91,16 +91,50 @@ void AMGe<dim, VectorType>::build_agglomerate_triangulation(
 {
   std::vector<typename dealii::DoFHandler<dim>::active_cell_iterator>
       agglomerate;
+  std::map<typename dealii::DoFHandler<dim>::active_cell_iterator,
```

Comment:
Add a comment describing what maps to what, "cell iterator -> (face index in cell, boundary id)".